docs/starter-kit-ui/starter_kit_ui.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: Overview
+title: Starter Kit UI Overview
```

Comment:
Please adjust the layout/title so that the sidebar label is "Configuring the UI", and further nesting (ie the Overview page) isn't necessary at this point, just have everything on "Configuring the UI" page